src/types/bchar.ml

```diff
@@ -0,0 +1,75 @@
+(* This file is free software, part of containers. See file "license" for more details. *)
+
+(** {1 Utils around char}
+
+    @since 0.14 *)
+
+include Char
+
+let equals (a:char) b = a=b
+let compare = Comparator.make Char.compare
+
+let lowercaseifAscii c =
+  if c >= 'A' && c <= 'Z'
+  then Char.unsafe_chr (Char. code c + 32)
+  else c
+
+let uppercaseifAscii c =
+  if c >= 'a' && c <= 'z'
+  then Char.unsafe_chr (Char.code c - 32)
+  else c
+
+let fromIntOrRaise = Char.chr
+let fromInt c = try Some (fromIntOrRaise c) with _ -> None
+let toInt = Char.code
+
+let isLowercase = function
```

Comment:
Can we implement String.blink() please? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/blink